src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ingroovy/StepInGroovy.java

```diff
@@ -0,0 +1,160 @@
+package org.jenkinsci.plugins.workflow.cps.steps.ingroovy;
+
+import groovy.lang.Closure;
+import org.apache.commons.io.FilenameUtils;
+import org.jenkinsci.plugins.workflow.cps.CpsVmThreadOnly;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.kohsuke.stapler.ClassDescriptor;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * A {@link Step} written entirely in Groovy.
+ *
+ * Unlike regular extensions, there will be one instance of {@link StepDescriptorInGroovy}
+ * for every step written in Groovy.
+ *
+ * @see 'doc/step-in-groovy.md'
```

Comment:
invalid javadoc comment. causes build failure.

To use a non class reference you must used a " double quote  - e.g. @see "doc/step-in-groovy.md"